AEPTarget/Sources/Target+PublicAPI.swift

```diff
@@ -392,4 +392,93 @@ import Foundation
             return
         }
     }
+
+    /// Retrieves Target response for an array of mbox locations.
+    ///
+    /// It issues a batch request to the configured Target server for the provided locations in the array. If the location name is not present for one or more items in the execute array, no execute request will be sent to Adobe Target.
+    ///
+    /// - Parameters:
+    ///   - executeArray: A dictionary containing notification data for a given location name.
+    ///   - completion: the callback which will be invoked with the Target response data or error message after the execute request is completed.
+    @objc(executeRawRequest:completion:)
+    static func executeRawRequest(_ executeArray: [[String: Any]], _ completion: @escaping ([[String: Any]]?, Error?) -> Void) {
+        if executeArray.isEmpty {
+            Log.error(label: LOG_TAG, "Failed to execute raw Target request, the provided execute array is empty.")
+            completion(nil, AEPError.invalidRequest)
+            return
+        }
+
+        var targetExecuteArray = [[String: Any]]()
+        for request in executeArray {
+            var mboxRequest = [String: Any]()
```

Comment:
Because isn't that how we init a collection, either
```
var mboxRequest: [String: Any] = [:]
```
or
```
var mboxRequest = [String: Any]()
```
Also see - https://forums.swift.org/t/initializing-an-empty-dictionary-style-question/11798

Reply:
true that both are valid.  i just like to follow apple's examples: https://docs.swift.org/swift-book/LanguageGuide/CollectionTypes.html